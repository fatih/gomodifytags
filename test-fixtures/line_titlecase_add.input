package foo

type foo struct {
	bar       string
	MyExample bool
	MyAnother []string
}
