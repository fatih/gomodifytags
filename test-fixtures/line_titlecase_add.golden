package foo

type foo struct {
	bar       string   `json:"Bar"`
	MyExample bool     `json:"My Example"`
	MyAnother []string `json:"My Another"`
}
